core/controllers/ingest.py

```diff
@@ -84,9 +89,10 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
         else:
             # TODO https://dluhcdigital.atlassian.net/browse/SMD-653: replace hardcoded dependencies with dependency
             #   injection
-            _ = extract_process_validate_tables(workbook_data, PF_TABLE_CONFIG)
+            user_tables = extract_process_validate_tables(workbook_data, PF_USER_DATA_TABLE_CONFIG)  # noqa: F841
+            control_tables = extract_process_validate_tables(workbook_data, PF_CONTROL_TABLE_CONFIG)
+            control_mappings = create_control_mappings(control_tables)  # noqa: F841
```

Comment:
Can we update the docstring to mention what these are and what they do, or leave an in-line comment?